scipy-1.0/paper.tex

```diff
@@ -884,8 +884,8 @@ \subsubsection*{Test suite}
 are more than \num{13000}~unit tests in the test suite, which is written for usage
 with the \texttt{pytest}\cite{pytest} framework.
 Test coverage at the SciPy~1.0 release point (Figure~\ref{fig:coverage}) was at
-87\% for Python code according to \texttt{pytest-cov}\cite{pytest-cov} and
-45\% for compiled (C, C++, and Fortran) code according to \texttt{gcov}\cite{gcov}.
+87\% for Python code according to \texttt{pytest-cov}\cite{pytest-cov}.
+Coverage for compiled (C, C++, and Fortran) code is still being implemented.
```

Comment:
Maybe something like this:

Coverage for compiled (C, C++, and Fortran) code was 45 %, or 59 % excluding most of the vendored (trusted external) and Cython-generated sources. Line coverage by a unit test does not guarantee correctness, and is not equivalent to formal proofing, but it can provide immediate feedback on changes in behavior.